cmd/lakectl/cmd/annotate.go

```diff
@@ -29,23 +28,36 @@ var annotateCmd = &cobra.Command{
 	Args:    cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		pathURI := MustParsePathURI("path", args[0])
+		recursive, _ := cmd.Flags().GetBool("recursive")
 		client := getClient()
 		pfx := api.PaginationPrefix(*pathURI.Path)
 		context := cmd.Context()
 		resp, err := client.ListObjectsWithResponse(context, pathURI.Repository, pathURI.Ref, &api.ListObjectsParams{Prefix: &pfx})
 		DieOnErrorOrUnexpectedStatusCode(resp, err, http.StatusOK)
+		const delimiter = "/"
```

Comment:
Is this necessary?

Reply:
Not really, but it is also used the same in other commands (`lakectl fs`)
I refactored it to be a const in common_helpers